x/zoneconcierge/keeper/ibc_packet_btc_headers.go

```diff
@@ -0,0 +1,67 @@
+package keeper
+
+import (
+	"context"
+
+	btclctypes "github.com/babylonlabs-io/babylon/x/btclightclient/types"
+	"github.com/babylonlabs-io/babylon/x/zoneconcierge/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// BroadcastBTCHeaders sends an IBC packet of BTC headers to all open IBC channels to ZoneConcierge
+func (k Keeper) BroadcastBTCHeaders(ctx context.Context) {
```

Comment:
Can we reuse `getHeadersToBroadcast` in this function, or merge these two functions to some extent?